docker/Dockerfile

```diff
@@ -1,18 +1,31 @@
 # syntax=docker/dockerfile:latest
 
 # Git: git version 2.30.2
-# /etc/os-release: Alpine Linux v3.16
+# /etc/os-release: Alpine Linux v3.20
 # Kernel: Linux ca3abedc4fb1 5.17.15-76051715-generic #202206141358~1655919116~22.04~1db9e34 SMP PREEMPT Wed Jun 22 19 x86_64 Linux
-# Build cmd: docker build --no-cache . -t nushell-latest
-# Other tags: nushell/alpine-nu:latest, nushell
+# Build cmd: docker build --no-cache . -t nushell:latest
+# Other tags: nushell:latest-alpine, nushell
 FROM alpine
 
-LABEL maintainer=nushell
-
 ARG TARGETARCH
 ARG ARCH=${TARGETARCH/arm64/aarch64}
+ARG ARCH=${ARCH/arm/armv7}
 ARG ARCH=${ARCH/amd64/x86_64}
 
+ARG BUILD_REF
+ARG BUILD_DATE
+LABEL maintainer="Nushell Team" \
```

Comment:
Seems like some of this stuff should match our cargo.toml file.
https://github.com/nushell/nushell/blob/2c31b3db072f60869dbe49f3bb1943d76c07b21e/Cargo.toml#L2-L14

We don't use the term "Nushell Team" specifically.

Reply:
I have just updated